### ISSUE TITLE
Can't verify gomuks session from Element

### ISSUE THREAD

**Issue by jjakob** [tulir/gomuks#425]
I'm having problems verifying my gomuks session from Element. I verified the Element session (which was the first session after registering) which reset cross-signing keys. I saved the recovery key to a file. Then I tried to verify gomuks with `/verify-device`. Everything works, I see the emojis on both sides and can confirm on both ends, gomuks says success but Element says the other device has cancelled verification. The gomuks session remains unverified.

I also tried both manual verification methods mentioned here #205 
- manually checking `/fingerprint` of gomuks in Element: I go to a room, find myself in the user list, find my session list, click on the gomuks session, the fingerprint does not match the one shown in gomuks.
- `/cs fetch` and `/cs self-sign`: fetch works (decrypted with the previously saved recovery key) but self-sign fails with: "Failed to self-sign: received update for device with different signing key"

**Comment by tulir** [comment]
You probably corrupted your session somehow, so you should log out and back in

**Comment by jjakob** [comment]
Right, that seems to have fixed it. I logged out, deleted gomuks data, logged back in, verification worked. Imported encryption keys from Element and I can see all messages.
I still have to see if other people can see my messages (before they couldn't, it said "waiting for message").
I don't know what the cause was but I saved the old gomuks folders before deleting them if there is any point in analyzing them?
